src/index.html

```diff
@@ -6,7 +6,40 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar calendar--length-29">
+      <!-- calendar__day--friday - modificator for calendar__day -->
```

Comment:
<!-- calendar__day--friday - modificator for calendar__day -->
you should implement this
```suggestion
```